examples/sui-move/escrow/sources/example.move

```diff
@@ -0,0 +1,510 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// An escrow for atomic swap of objects that trusts a third party for liveness,
```

Comment:
If I understand correctly, this is an example from single-writer objects. And it would be great if we could mention it.